RELEASE.txt

```diff
@@ -64,6 +69,17 @@ FIXES
 - Ninja: Fix execution environment sanitation for launching ninja. Previously if you set an
   execution environment variable set to a python list it would crash. Now it
   will create a string joining the list with os.pathsep
+- Inconsistent behavior of adding CPPDEFINES values via tuple arguments
```

Comment:
Since this can change the CPPDEFINES, we should add a section at the top with
NOTE: The following usages of CPPDEFINES have changed
env.Append(CPPDEFINES,('a','b') )  now yields -Da -Db instead of -Da=b
etc..
Should be a fairly small set of these to clearly spell out what's changed?